google/cloud/bigquery/job/load.py

```diff
@@ -442,6 +442,10 @@ def time_partitioning(self):
         Only specify at most one of
         :attr:`~google.cloud.bigquery.job.LoadJobConfig.time_partitioning` or
         :attr:`~google.cloud.bigquery.job.LoadJobConfig.range_partitioning`.
+        
+        .. note::
+
+           You can't create partitions in already existing unpartitioned tables.
```

Comment:
A link to reference / cloud.google.com docs describing this restriction would be appreciated.

Reply:
I got an error message when trying to do this. Can't find anything in the docs. It should be 
here https://cloud.google.com/bigquery/docs/partitioned-tables#limitations 
or here https://cloud.google.com/bigquery/docs/creating-partitioned-tables